convert-image-format@el-amine-404/info.json

```
@@ -0,0 +1,3 @@
+{
+  "author": "amine-el-houjjaji"
```

```suggestion
  "author": "el-amine-404",
```
This should be your GitHub username, matching the UUID.